<?xml version="1.0"?>
<launch>
	<arg name="topic" default="usb_camera" />
	<arg name="device_path" default="/dev/video_front"   />
	<arg name="framerate" default="-1"  />
	<node name="$(anon usb_camera)" pkg="cv" exec="usb_camera" output="screen">
		<param name="topic" value="$(var topic)" />
		<param name="device_path" value="$(var device_path)" />
		<param name="framerate" value="$(var framerate)" />
	</node>
</launch>
